docs/api/rest_api/rest_api_reference/input/ibexa-types.raml

```diff
@@ -2332,7 +2328,7 @@ ContentTypeCreate:
             description: 'If set, this value overrides the current user as creator.'
             type: any
         creationDate:
-            type: DateTime
+            type: datetime
```

Comment:
Can a timezone be passed?
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#date
```suggestion
            type: datetime-only
```